### PR TITLE
git statusCreated the frontend for the surveyor, submit needs to be added still

### PR DESCRIPTION
**Pull request by nickcoutts99**

### CODE REVIEW

File: src/main/resources/static/surveyor.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#question {
+    display: grid;
+    border-radius: 15px;
+    border: 2px solid #757575;
+    padding: 20px;
```
Comment (rafiddewan) [inline src/main/resources/static/surveyor.css:5]: Minor optional suggestion but you can do the padding with percentages instead to make it scalable: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

File: src/main/resources/static/surveyor.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+$(document).ready(function() {
+    //Add's new multiple choice question
+    $(".new-MCQ").click(function(){
+        var numOptions = Math.floor($("#MCQOptionsNum").val());
+        var newMCQ = `<div class='multipleChoiceQuestion' id="question">
+                        <label for='QuestionText'>Question:</label>
+                        <input type='text' id='QuestionText'>`;
+        if(numOptions < 1){
+            alert("Number of Multiple Choice Options must be greater than 0!");
```
Comment (rafiddewan) [inline src/main/resources/static/surveyor.js:9]: an MCQ usually has minimum two options (true, false)

Comment (rafiddewan) [reply src/main/resources/static/surveyor.js:9]: Should be "greater than 1!" in the error message

File: src/main/resources/static/surveyor.css (hunk 1 above)
Comment (nickcoutts99) [reply src/main/resources/static/surveyor.css:5]: everywhere else we are just using pixels, i wouldnt stress too much about scalability. We arent panning for ios/mobile users to use this. So i think this is fine

File: src/main/resources/static/surveyor.js (hunk 2 above)
Comment (nickcoutts99) [reply src/main/resources/static/surveyor.js]: updated it.

**Review by rafiddewan — APPROVED** [review]

**Pull request merged**